opennlp-tools/src/main/java/opennlp/tools/formats/BioNLP2004NameSampleStream.java

```diff
@@ -32,15 +32,25 @@
 import opennlp.tools.util.StringUtil;
 
 /**
- * Parser for the training files of the BioNLP/NLPBA 2004 shared task.
+ * A {@link ObjectStream sample stream} for the training files of the
+ * BioNLP/NLPBA 2004 shared task.
  * <p>
- * The data contains five named entity types: DNA, RNA, protein, cell_type and cell_line.<br>
+ * The data contains five named entity types:
+ * <ul>
+ *   <li>{@code DNA}</li>
+ *   <li>{@code RNA}</li>
+ *   <li>{@code protein}</li>
+ *   <li>{@code cell_type}</li>
+ *   <li>{@code cell_line}</li>
+ * </ul>.
  * <p>
- * Data can be found on this web site:<br>
- * http://www-tsujii.is.s.u-tokyo.ac.jp/GENIA/ERtask/report.html
+ * Data can be found on
+ * <a href="http://www-tsujii.is.s.u-tokyo.ac.jp/GENIA/ERtask/report.html">this website</a>
```

Comment:
Link is down for me

Reply:
Alternatives?

Reply:
It looks like this repo contains a copy of the data located on the original page: https://github.com/spyysalo/jnlpba
The BioNLP 2004 seems to be related to http://www.geniaproject.org/shared-tasks/bionlp-jnlpba-shared-task-2004

Reply:
Resolved in next commit.